### PR TITLE
calculate percentiles during bootstrapping phase

### PR DESCRIPTION
**Pull request by yashlyft**

### CODE REVIEW

**Comment by yashlyft** [comment]
meow @theatrus

File: src/config.c [hunk 1]
```diff
@@ -167,6 +170,25 @@ static int value_to_list_of_doubles(const char *val, double **result, int *count
...
 
+static int calculate_percentiles(double *quantiles, int **result, int count) {
+    int percentile;
+    double quantile;
+
+    *result = (int*)malloc(count * sizeof(int));
+
+    // make sure we have enough memory
+    for (int i = 0; i < count; i++) {
+       quantile = *(quantiles + i);
```
Comment (theatrus) [inline src/config.c]: Probably just use the traditional: `quantiles[i]`

Comment (theatrus) [inline src/config.c]: Probably use the normal `quantiles[i]` form

**Comment by theatrus** [comment]
Whoops sorry about the double submit

**Pull request merged**